test/integration/crud/bulk.test.ts

```diff
@@ -104,6 +104,125 @@ describe('Bulk', function () {
           }
         });
       });
+      context('BulkWriteResult should not include invalid insert in insertedIds', function () {
+        async function insertManyTryInvalidIds(input, isOrdered, indices, expectedInsertedIds) {
```

Comment:
We're not testing invalid ids here, but duplicate fields on unique indexes.

```suggestion
        async function assertFailsWithDuplicateFields(input, isOrdered, indices, expectedInsertedIds) {
```